emoji-ui/src/__tests__/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "globals": {
+    "it": "readonly"
```

Comment:
what does this lint config do?

Reply:
it lets the tests use "it" as a global variable. there's probably some other way of doing it but i didn't want to spend time configuring lint.

Reply:
at least, i didn't want to spend any more time configuring lint than i needed to to get it to find basic syntax errors for me.